docs/src/main/asciidoc/lra-guide.adoc

```diff
@@ -0,0 +1,196 @@
+The LRA (short for Long Running Action) participant extension is useful in microservice based
+designs where different services can benefit from a relaxed notion of distributed consistency.
+
+The idea is for multiple services to perform different computations/actions in concert, whilst
+retaining the option to compensate for any actions performed during the computation.
+This kind of loose coupling of services bridges the gap between strong consistency models
+such as JTA/XA and "home grown" ad hoc consistency solutions.
+
+The model is based on the https://github.com/eclipse/microprofile-lra/blob/master/spec/src/main/asciidoc/microprofile-lra-spec.adoc#eclipse-microprofile-lra[Eclipse Microprofile LRA specification].
+The approach is for the developer to annotate a business method with a Java annotation
+(https://github.com/eclipse/microprofile-lra/blob/master/api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java[`@LRA`]).
+When such a method is called a context is created (if one is not already present) which is passed
+along with subsequent JAX-RS invocations until a method is reached
+which also contains an `@LRA` annotation with an attribute that indicates that the LRA should be
+closed or cancelled. The default is for the LRA to be closed in the same method that started the
+LRA (which itself may have propagated the context during method execution).
+The JAX-RS resource indicates that it wishes to participate in the interaction by, minimally,
+marking one of the methods with an
+https://github.com/eclipse/microprofile-lra/blob/master/api/src/main/java/org/eclipse/microprofile/lra/annotation/Compensate.java[`@Compensate`]
+annotation. If the context is later cancelled then this compensate action is guaranteed to be
+called even in the presence of failures and is the trigger for the resource to compensate for any
+activities it performed in the context of the LRA. This guarantee enables services to operate
+reliably with the assurance of eventual consistency (when all compensation activities have
+ran to completion).
+
+Note that this version of the extension only supports JAX-RS based services (and a subsequent
+revision will include support for non JAX-RS based services).
+
+== Configuration
+
+Once you have your Quarkus maven project configured you can add the `narayana-lra` extension
+by running the following command in your project base directory:
+
+[source,bash]
+----
+./mvnw quarkus:add-extension -Dextensions="narayana-lra"
+----
+
+This will add the following to your pom.xml
+
+[source,xml]
+----
+    <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-narayana-lra</artifactId>
+    </dependency>
+----
+
+If there is a running coordinator then this is all you need in order to create
+new LRAs and to enlist participants with them.
+
+The LRA extension can be configured by placing an `application.properties` file
+in the `src/main/resources` directory. The only LRA specific property is
```

Comment:
This file is automatically created when you create the Quarkus project.

Reply:
I could reword it as `by updating the` instead of `by placing an`. I have never used any tool to `create the Quarkus project`, I just write the pom like I would write any java application. Do you have a feel for how many people use the tool you are referring to?

Reply:
Most users use code.quarkus.io. Users writing the pom.xml themselves are generally more advanced and so now when (and where) to create the file. So, +1 to reword as you proposed.